go/test/endtoend/recovery/pitr/shardedpitr_test.go

```diff
@@ -0,0 +1,539 @@
+/*
```

Comment:
Because you renamed the file, I can't tell what changed. Can you rename this to pitr_test.go?

Reply:
The codeflow of current testcase is totally different from the previous testcase. As there will be a lot of diff between the previous testcase and current testcase, I kept a different filename. 

If required, I will change the filename back to pitr_test.go